### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ColorContrast:The ratio of the test is below 4.5:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9297]
[“Check out Accessibility Insights! -](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.4.3;#ColorContrast;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tool:Accessibility insights for desktop
MAS References:MAS1.4.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.
8. Launch accessibility insights for desktop and select third button "Contrast"

**Actual:**
When the color contrast of the text "information" and the text under it as well as the text copy and close are below 4.5:1 ratio.

**Expected:**
The ratio should be 4.5:1 or greater.

**User Impact:**
Low vision users will face difficulty in viewing the content.

**Attachment for Reference :**
![image](https://user-images.githubusercontent.com/24580570/75150276-0b6a9a00-572a-11ea-9da7-c160c0e90944.png)

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
